lib/BUILD.bazel

```diff
@@ -1,6 +1,6 @@
 cc_library(
     name = "mbedtls_config",
-    hdrs = ["include/mbedtls_config.h"],
+    hdrs = ["include/mbedtls_config.h", "include/picotool_mbedtls_config.h"],
```

Comment:
Does it make sense to list **both** of these files, when `mbedtls_config.h` is merely a wrapper around `picotool_mbedtls_config.h` ? Or is this a required backwards-compatibility-thing in order not to break any existing Bazel projects? (I'm afraid I don't know much about Bazel!)